### PR TITLE
Milestone 5: Views

### PR DESCRIPTION
**Pull request by medaminedev66**
- Implement the design from the sneak peek wireframes.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+</div>
+<div class="bio">
+  <p class="fs-4">Bio</p>
+  <p><%= @user.Bio %></p>
+</div>
+<% @user.recent_posts.each do |post| %>
+<div class="post">
+  <p class="fs-5">Post <%= post.id %> </p>
+  <p><%= post.Text %> </p>
+</div>
```
Comment (DeliceLydia) [inline app/views/users/show.html.erb:19]: - [ ] On the user page you should display only the 3 most recent posts and then here you forgot to add 'see all post' link to be able to see all user posts, please add that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
### Hi @medaminedev66, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (medaminedev66) [reply app/views/users/show.html.erb:19]: Hi @DeliceLydia, The method that I used here `recent_posts` shows only 3 posts.

Comment (medaminedev66) [reply app/views/users/show.html.erb:19]: Ok I will add `see all posts` button

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Amine, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (rloterh) [reply app/views/users/show.html.erb:19]: This issue is well resolved! ✔️

**Pull request merged**